### DISCUSSION TITLE
Fetch an http:// API

### DISCUSSION THREAD

**Discussion by eli-ott** [vercel/community#1385] [Help] [answered]
### Summary

I need to fetch an API that's only available on http:// but vercel doesn't authorize it. I was wondering if there's any way to bypass that ?

### Additional information

Here's what I get in the console: `Mixed Content: The page at '<URL>' was loaded over HTTPS, but requested an insecure resource '<URL>'. This request has been blocked; the content must be served over HTTPS.`
The API is [http://api.open-notify.org/iss-now.json](http://api.open-notify.org/iss-now.json)

### Example

https://iss-tracker-liart.vercel.app/

**Comment by swarnava** [comment, marked as answer]
Hi there,

We don't support non https URL in Vercel platform. Let us know if there is anything we can help you!

**Reply by habibir7** [reply]
i encountered same problem when vercel  refused connection because my api are http://
